common/src/main/java/org/apache/rocketmq/common/topic/TopicValidator.java

```diff
@@ -57,13 +85,24 @@ public static boolean validateTopic(String topic, RemotingCommand response) {
             return false;
         }
 
-        //whether the same with system reserved keyword
-        if (topic.equals(MixAll.AUTO_CREATE_TOPIC_KEY_TOPIC)) {
+        if (isSystemTopic(topic)) {
             response.setCode(ResponseCode.SYSTEM_ERROR);
-            response.setRemark("The specified topic is conflict with AUTO_CREATE_TOPIC_KEY_TOPIC.");
+            response.setRemark("The topic[" + topic + "] is conflict with system topic.");
             return false;
         }
```

Comment:
[Important] There is a serious problem. msgCheck call this method and check all system topics, this will cause the client to fail to send messages to **all system topics**, but it is necessary for the client to send messages to certain system topics, such as BenchmarkTest, OFFSET_MOVED_EVENT.  IMO, We need a blacklist to prohibit sending to certain system topics, but not all system topics.

Reply:
This is really a deadly problem. Schedule Topic is the only one should prevent send by user,I’ll check the method TopicValidator#validateTopic everywhere invoked.